devel/lib/lib.sh

```diff
@@ -81,5 +85,19 @@ require_image() {
   bazel run --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 "${BAZEL_TARGET}"
 
   # Load the image into the kind cluster
-  kind load docker-image --name "$KIND_CLUSTER_NAME" "$IMAGE_NAME"
+  load_image "$IMAGE_NAME"
 }
+
+# load_image will lod an image into the local cluster
+# for a kind cluster it will load it into the cluster
+# with name $KIND_CLUSTER_NAME
+load_image() {
+  IMAGE_NAME="$1"
+  if [[ "${IS_OPENSHIFT:-}" == "true" ]] ; then
```

Comment:
We reference `IS_OPENSHIFT` here in lib.sh but don't name it/export it at the top - moving it there should hopefully simplify things quite a bit and mean we can 'expect' it to be set here too 😄 (and fail if it is not for some reason)